app/front-end/src/features/editor/components/toolbarView/toolbarGroupParams/applyGroupParams.tsx

```diff
@@ -0,0 +1,190 @@
+                  </StyledGroupParamsListSubheader>
+                );
+              }
+
+              return (
+                <StyledGroupParamsMenuItem key={file.id} value={file.id}>
+                  <StyledGroupParamsMenuItemTypography>{file.label}</StyledGroupParamsMenuItemTypography>
+                </StyledGroupParamsMenuItem>
+              );
+            })}
+          </StyledGroupParamsSelect>
+        </FormControl>
+      </Box>
+      <Box sx={{ height: '100%', display: 'flex', flexDirection: 'column', alignItems: 'end', gap: '0.5rem' }}>
+        <FormControl sx={{ width: '90%' }}>
+          <StyledGroupParamsInputLabel
+            sx={{ color: blocked ? Theme.palette.action.disabled : Theme.palette.text.primary }}
+          >
+            Save To
+          </StyledGroupParamsInputLabel>
+          <StyledGroupParamsSelect
+            id={'save-to-select'}
+            name={'save-to'}
+            label={'Save To'}
+            value={saveToValue}
+            onChange={handleSaveToChange}
+            disabled={blocked}
+            MenuProps={{
+              PaperProps: {
+                sx: {
+                  maxWidth: '15.7rem',
+                  maxHeight: '20rem',
+                  bgcolor: Theme.palette.background.default,
+                },
+              },
+            }}
+          >
+            <StyledGroupParamsMenuItem key={'new-file'} value={'/'}>
+              <StyledGroupParamsMenuItemTypography>New file...</StyledGroupParamsMenuItemTypography>
+            </StyledGroupParamsMenuItem>
+            <StyledGroupParamsListSubheader key={'root'}>
+              <StyledGroupParamsMenuItemTypographyBold>root:</StyledGroupParamsMenuItemTypographyBold>
+            </StyledGroupParamsListSubheader>
+            {fileArray.map((file) => {
+              if (file.type === FileTypes.FOLDER) {
+                return (
+                  <StyledGroupParamsListSubheader key={file.id}>
+                    <StyledGroupParamsMenuItemTypographyBold>{file.id}:</StyledGroupParamsMenuItemTypographyBold>
+                  </StyledGroupParamsListSubheader>
+                );
+              }
+
+              return (
+                <StyledGroupParamsMenuItem key={file.id} value={file.id}>
+                  <StyledGroupParamsMenuItemTypography>{file.label}</StyledGroupParamsMenuItemTypography>
+                </StyledGroupParamsMenuItem>
+              );
+            })}
+          </StyledGroupParamsSelect>
+        </FormControl>
+        <FormControlLabel
+          control={
+            <Checkbox
+              id='override-checkbox'
+              checked={overrideValue}
+              onChange={handleOverrideChange}
+              disabled={blocked}
+            />
+          }
+          label={
+            <StyledGroupParamsTypography sx={{ color: blocked ? Theme.palette.action.disabled : 'initial' }}>
+              Override
```

Comment:
It would also be nice to rename 'Override' to 'Override file' for clarity

Reply:
Fixed with [MDE/PKFE-31 params additional fixes and changes](https://github.com/Strexas/kath/pull/57/commits/6c8fad426f15d0b9d6eec084d13a931512fcfb2d)